src/lib/metrics.ts

```diff
@@ -258,6 +259,12 @@ export default class MetricsMonitor {
             help: 'Duration of mapFeaturesForClient function',
         });
 
+        const projectEnvironmentsDisabled = createCounter({
+            name: 'project_environments_disabled',
+            help: 'How many "environment disabled" events we have received for each project',
+            labelNames: ['project_id'],
```

Comment:
Hmm. I see that we have multiple variants of label names in here. On line 97 we use camelCase but on the sdk version counter we use snake case. I wish we kept to one convention, but I guess that's too late now. The question is which convention do we want to stick with for the future?

Reply:
Good point! I chose snake case for two reasons:
1. I didn't see line 97. I stopped when I found the ones for sdk_version
2. It appears that the rest of prometheus (e.g. the name of the metric) uses snake case ("project_environments_disabled"), so I figured we should be consistent in metric names and label names.

And on that last point, I suggest that we stay with snake_case so that metric names and label names use the same format.